src/main/java/io/axoniq/axonserver/connector/admin/impl/AdminChannelImpl.java

```diff
@@ -142,32 +142,58 @@ protected EventProcessor terminalMessage() {
 
     @Override
     public CompletableFuture<Result> pauseEventProcessor(String eventProcessorName, String tokenStoreIdentifier) {
```

Comment:
I would move this to a default implementation in the interface so that other implementations for the interface would not have to implement this. 
Also, use a constant for the empty string here, e.g. CHANNEL_CONTEXT, to clarify what this parameter means.